yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -520,7 +529,12 @@ func gatherAssessmentMetadataFromOracle() (err error) {
 	}
 
 	log.Infof("using script: %s", scriptPath)
-	return runGatherAssessmentMetadataScript(scriptPath, []string{"ORACLE_PASSWORD=" + source.Password},
+	tnsAdmin, err := getTNSAdmin(source)
+	if err != nil {
+		return fmt.Errorf("error getting tnsAdmin: %v", err)
+	}
+	log.Infof("tnsAdmin passed to oracle gather metadata script: %s", tnsAdmin)
+	return runGatherAssessmentMetadataScript(scriptPath, []string{"ORACLE_PASSWORD=" + source.Password, "TNS_ADMIN=" + tnsAdmin},
```

Comment:
I would suggest:
1. `runGatherAssessmentMetadataScript` should start with `env = os.Environ()` and then append whatever you want to append. We do that with all other commands that we run.
2. For TNS_ADMIN env var, makes sense what you're doing. Let's explicitly retrieve it and pass it. 
2. We should do the same for ORACLE_HOME. The priority should be oracleHome voyager flag, env var ORACLE_HOME, default `/usr/lib/oracle/21/client64`. 

(Something for the future: I don't see why we need to have oracle-home as a flag. We should just pick up the env var `ORACLE_HOME`)

Reply:
ok, i believe after that we won't need to pass the `PATH` variable separately.

Reply:
doing the change